### ISSUE TITLE
Add paging for query results

### ISSUE THREAD

**Issue by joewiz** [eXist-db/atom-existdb#56]
Currently query results are truncated to 10 items, with no mechanism to page through additional results. 

Adding previous & next commands would be great. Exposing these via toolbar (see #53) would natural.

Adding a setting for the number of results per page would also be convenient.

**Comment by duncdrum** [comment]
actually I think the way query results are handled is in need of an overhaul. There should not be a `query-results` file that appears in the repo in certain conditions, but not others. 
But first we need @wolfgangmm to add his token to travis ;-)

**Comment by duncdrum** [comment]
yes and sometime they are written to a tmp location and sometimes into the code repo I'm currently working on

**Comment by joewiz** [comment]
Without you hitting save? Reproducible? Sounds like a bug - sufficiently different from the issue here that it’s worth its own issue.
